src/views.rs

```diff
@@ -924,6 +924,15 @@ pub struct EncodableVersion {
     /// inside it.
     #[schema(value_type = Option<Object>)]
     pub trustpub_data: Option<TrustpubData>,
+
+    /// Line count statistics for this version.
+    ///
+    /// Status: **Unstable**
+    ///
+    /// This field may be `null` until the version has been analyzed, which
+    /// happens in an asynchronous background job.
+    #[schema(value_type = Object)]
+    pub linecounts: Option<serde_json::Value>,
```

Comment:
I wonder about calling this `experimental_linecounts` or `unstable_linecounts` to make that status clear, even for API users that don't know about the OpenAPI documentation 🤔

Reply:
> I wonder if we should say a little more about the shape of this field in the documentation, since it's just going to appear as an `Object` in the API.

I explicitly didn't want to document the exact content yet while this is being considered unstable 😅 

I think once we stabilize this we should document it, but as long as we're the only users this might actually be counter-productive